src/Geta.Epi.MediaReferenceSelector/Geta.Epi.MediaReferenceSelector.nuspec

```diff
@@ -13,13 +13,15 @@
     <copyright></copyright>
     <tags>EPiServer</tags>
     <dependencies>
-      <dependency id="EPiServer.CMS.Core" version="10.2.0" />
-      <dependency id="EPiServer.CMS.UI.Core" version="10.2.0" />
+      <dependency id="EPiServer.CMS.Core" version="[11.3.2, 12)" />
```

Comment:
Why not have a lower start version? 11.1 for instance.

Reply:
Because I only tested it with version 11.3.2. Do you think it's safe to specify a lower version?

Reply:
If you go lower you need to also update the references that this package will use when compiling to the same lower version. As a rule we try to always go for the first version of a major release.

Reply:
That make sense. Downgraded to version 11.1.0 and added changelog.md